blocks/inspector-controls/radio-control/index.js

```diff
@@ -32,7 +32,7 @@ function RadioControl( { label, selected, help, instanceId, onChange, options =
 						name={ id }
 						value={ option.value }
```

Comment:
I think we can safely remove the value prop here, can we?

Reply:
I tried removing the value prop but the radio button never gets selected so it seems to be required.

Reply:
You're right! I don't know what happened in my testing 🙃